tests/generic_relations/tests.py

```diff
@@ -259,10 +259,11 @@ def test_queries_content_type_restriction(self):
             Animal.objects.filter(tags__tag="fatty"),
             [self.platypus],
         )
-        self.assertSequenceEqual(
-            Animal.objects.exclude(tags__tag="fatty"),
-            [self.lion],
-        )
+        # Exists is not supported in MongoDB
```

Comment:
I added this change to the fork. I think that's all this branch requires, but let me know.

Reply:
Indeed, if we aren't using objectIdField we just need to modify this test only.